<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encryption tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Password">
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_showPassword">
          <property name="text">
           <string>Show</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_confirmPassword">
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_mode">
          <item>
           <property name="text">
            <string>GCM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CBC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CTR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OCB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EAX</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SIV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CCM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CFB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OFB</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_cipher">
          <item>
           <property name="text">
            <string>AES</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ChaCha20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kuznyechik</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Camellia</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SM4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Serpent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Twofish</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Threefish-512</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SHACAL2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blowfish</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IDEA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3DES</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Encrypt">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Encrypt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Decrypt">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Decrypt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_header">
          <property name="text">
           <string>Header</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="checkBox_chain">
          <property name="text">
           <string>Chain</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addCipher">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_removeCipher">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_cipherList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_inputFile">
          <property name="text">
           <string>Input file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_inputFile">
          <property name="text">
           <string>Drag and drop file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QComboBox" name="comboBox_PBKDF">
          <item>
           <property name="text">
            <string>Argon2id</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Argon2d</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Argon2i</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scrypt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PBKDF2</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_memcost">
            <property name="text">
             <string>Memcost (MiB)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_memcost">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>2048</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_iterations">
            <property name="text">
             <string>Iterations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_timecost">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Threads</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_threads">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>453</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
